### PR TITLE
feat: Added Elm preset

### PR DESCRIPTION
**Pull request by mblarsen**
A language preset for Elm

* Hot reloading
* Option to turn on/off Elm debugger (aka time traveller)
* Includes working example

### CODE REVIEW

File: packages/poi-preset-elm/index.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    poi.extendWebpack(config => {
+      config.module
+        .rule('elm')
+          .test(/\.elm$/)
+          .exclude.add([/elm-stuff/, /node_modules/]).end()
+          .use('elm-hot')
+            .loader('elm-hot-loader').end()
+          .use('elm-webpack')
+            .loader('elm-webpack-loader')
+              .options(poi.options.mode === 'production' ? {} : Object.assign(optionDefaults, options || {}))
```
Comment (egoist) [inline packages/poi-preset-elm/index.js]: why do we only set `options` in non-production mode?

Comment (mblarsen) [reply packages/poi-preset-elm/index.js]: Hmm, yeah we do want some of them. Committing a change that will only disable debug and leave the others as is.

Comment (mblarsen) [reply packages/poi-preset-elm/index.js]: I appreciate how thorough you are with these reviews ;)

Comment (mblarsen) [reply packages/poi-preset-elm/index.js]: Hmm.. odd. New commits not showing up.

Comment (mblarsen) [reply packages/poi-preset-elm/index.js]: Only when I force pushed they appeared.

**Comment by mblarsen** [comment]
> Partially Degraded Service - components affected: GitHub

Wonder if it will rerun once service is back.

**Pull request merged**